src/components/NewWorkspaceModal.js

```diff
@@ -154,13 +154,20 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
       ])]),
       h(IdContainer, [id => h(Fragment, [
         h(FormLabel, { htmlFor: id, required: true }, ['Billing project']),
+        div({ style: { margin: '0.5rem 0' } }, ['A billing project must have a billing account which Terra has access to in order to be used.']),
         h(Select, {
           id,
           isClearable: false,
           placeholder: 'Select a billing project',
           value: namespace,
           onChange: ({ value }) => setNamespace(value),
-          options: _.uniq(_.map('projectName', billingProjects)).sort()
+          options: _.uniq(_.map(({ projectName, invalidBillingAccount }) => {
+            return {
+              label: projectName,
+              value: projectName,
+              isDisabled: invalidBillingAccount
```

Comment:
this might not be an issue with this PR though, since the DB is not showing the billing accounts either.

```
> select * from BILLING_PROJECT where NAME = 'aj-bp-test-1owner' \G
*************************** 1. row ***************************
                               NAME: aj-bp-test-1owner
                CROMWELL_BUCKET_URL: gs://cromwell-auth-aj-bp-test-1owner
                    CREATION_STATUS: Ready
                    BILLING_ACCOUNT: null
                            MESSAGE: NULL
                   CROMWELL_BACKEND: PAPIv2
                  SERVICE_PERIMETER: NULL
              GOOGLE_PROJECT_NUMBER: 576518341068
               INVALID_BILLING_ACCT:
               SPEND_REPORT_DATASET: NULL
                 SPEND_REPORT_TABLE: NULL
SPEND_REPORT_DATASET_GOOGLE_PROJECT: NULL
1 row in set (0.064 sec)
```

even though the UI shows a valid billing account
![Screen Shot 2021-09-15 at 9 51 34 AM](https://user-images.githubusercontent.com/5375000/133445982-fd8bdfea-ae2e-43a3-a5cd-75fbe1752bfc.png)

Reply:
^ not sure why the UI would show a billing account if the DB says the billing account is `null` though

Reply:
Dev's UI checks google directly, whereas the create needs it in our DB. v2_workspaces branch on the UI does the right thing.